packages/ui-components/test/toolbar.spec.ts

```diff
@@ -294,6 +294,7 @@ describe('@jupyterlab/ui-components', () => {
         const buttonNode = button.node.firstChild as HTMLButtonElement;
         expect(buttonNode.disabled).toBe(true);
         expect(buttonNode.classList.contains('lm-mod-toggled')).toBe(true);
```

Comment:
Do we still need to check the `lm-mod-toggled` class ?
As far as I understand the style is now applied on `aria-pressed` ?

Reply:
I kept the test to check we are backward compatible for extensions using that class for styling.